### ISSUE TITLE
Radio button fails axe test due to role="presentation" on input

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-radio-button#139]
The following output is produced when running `axe` tests for the next generation version.

```sh
Error: Accessibility Violations
---
Rule: aria-allowed-role
Impact: minor
ARIA role must be appropriate for the element (https://dequeuniversity.com/rules/axe/3.5/aria-allowed-role?application=axeAPI)

Issue target: vaadin-radio-button,input
Context: <input type="radio" role="presentation" tabindex="0">
Fix any of the following:
  ARIA role presentation is not allowed for given element
```

We have this problem with `vaadin-button` also, see vaadin/vaadin-button#144

**Comment by web-padawan** [comment]
Here is the current list of [allowed elements](https://github.com/dequelabs/axe-core/blob/ad1448088b93e7e9bb51a5f854234266c1e98c51/lib/commons/aria/index.js#L1483-L1523) that can use `presentation` role according to `axe`.

**Comment by web-padawan** [comment]
See https://www.w3.org/TR/using-aria/#4thrule

**Comment by web-padawan** [comment]
One more note: `vaadin-radio-button` was created based on `vaadin-checkbbox`.
This is the commit that added this role: vaadin/vaadin-checkbox@e8e11cc2c104060cfd8f3384961922e45841dde4
